### DISCUSSION TITLE
sLAQA parameters and examples

### DISCUSSION THREAD

**Discussion by ypanhtu** [Tomoki-YAMASHITA/CrySPY#69] [Q&A]
Dear Dr. Yamashita,

I have not found any parameters and examples of the sLAQA method in the website of CrySPY (https://tomoki-yamashita.github.io/CrySPY_doc/index.html) which was introduced in your great paper npj Comput. Mater. (2018) 4:32.

So, would you like introduce the parameters and share any examples about the sLAQA method.

Thanks a lot.

Roc 

HTU

**Comment by Tomoki-YAMASHITA** [comment]
CrySPY does not have the "sLAQA" mode because the number of structures in the candidate pool can be increased at any time (but manually).
If you want to increase the total number of structures (num. of initial structures), just increase `tot_struc` in the middle of the simulation.

The examples of LAQA can be found in CrySPY utility (https://github.com/Tomoki-YAMASHITA/CrySPY_utility/tree/master/examples).
The laqa score is adopted from Eq. 2 in this paper (https://www.tandfonline.com/doi/full/10.1080/27660400.2022.2059335).
I think we should use the default weight parameters for now. (https://tomoki-yamashita.github.io/CrySPY_doc/input/laqa/index.html) 
The value of `nselect_laqa` depends on the computing resources you have.

If you have any other questions, please feel free to ask.

**Reply by ypanhtu** [reply]
Thank you Dr. Yamashita.  I  get it.
